eos-tech-support/eos-hack-reboot-to-first-contact

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+usage () {
+    cat <<EOF
+Usage: $0
+Reboot the machine and deletes all users in the system,
+so you can execute the first contact experience from
```

Comment:
I would just name this something like eos-user-reset or eos-erase-users similar, as it's not specific to Hack and could be a useful CS tool generally.